qemu/tests/win_virtio_driver_installer_uninstall.py

```diff
@@ -86,8 +87,9 @@ def run(test, params, env):
     # viostor and vioscsi drivers can not uninstalled by installer
     for device_name in device_name_list:
         chk_cmd = params["vio_driver_chk_cmd"] % device_name[0:30]
-        status = session.cmd_status(chk_cmd)
-        if status == 0:
+        output = session.cmd_output(chk_cmd).strip()
+        inf_name = re.findall(r"\.inf", output, re.I)
```

Comment:
Why we need `re.I` here?

Reply:
Are you sure the output is all lowercase on all windows guests and later?

Reply:
I am not sure.I just want to confirm why we need it?Do we know which windows OS will different?If not, I think we should not add re.I here.

Reply:
I don't know.
@vivianQizhu  Could you check it?

Reply:
> I am not sure.I just want to confirm why we need it?Do we know which windows OS will different?If not, I think we should not add re.I here.

I don't know why you recommend to delete it, it affects nothing.

Reply:
First, it impacts nothing.
Second, there is no need to do code update later if some windows os returns the uppercase.

Reply:
The expected str is` .inf`, but in this patch, `.INF/.inF/...` will also pass the check,I think we need a reason to use re.I.
If all windows version output is lowercase, we should not add it here.

Reply:
.INF/.inF/... will also pass the check
I don't understand what you mean.

Reply:
Actually I think remove it or not , both of them are okay, we shouldn't waste time on the tiny thing.